tests/draft3/optional/format/date.json

```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "description": "validation of date strings",
+        "schema": {"format": "date"},
+        "tests": [
+            {
+                "description": "a valid date string",
+                "data": "1963-06-19",
+                "valid": true
+            },
+            {
+                "description": "an invalid date string",
+                "data": "06/19/1963",
+                "valid": false
+            }
+        ]
+    }
+]
```

Comment:
It looks like the draft 3 version is inconsistent with the other ones (still lives in tests/draft3/optional/format/ecmascript-regex.json -- I'd comment on that file but GH is wonderful and doesn't let you comment on empty diffs).

Reply:
fixed.